<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>LogFileMaker for YU Blueprint BlackCreek DBMS Project</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Misc/imgs/blueprint.jpg</normaloff>../../Misc/imgs/blueprint.jpg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 77, 156);</string>
  </property>
  <widget class="QLabel" name="LogFileMakerTitle">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
background-color: rgb(255, 170, 0);
border: 1px solid black;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>LogFileMaker</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HowManyFilesLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>How many files? (Each file is 1MB): </string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleDescription">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>55</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A tool made by Ryan Mehr for the YU Blueprint BlackCreek DBMS project.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="numFilesSlider">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TickPosition::TicksBothSides</enum>
   </property>
  </widget>
  <widget class="QLabel" name="numFilesCurValLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current value: 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearInputsBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 214, 7);
border: 1px solid black;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Clear Inputs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteLogFilesBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 214, 7);
border: 1px solid black;
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>Delete Log Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.942105 rgba(255, 122, 0, 255));</string>
   </property>
   <property name="text">
    <string>Create files!</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusText">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>450</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>STATUS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusValueLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Undetermined.</string>
   </property>
  </widget>
  <widget class="QLabel" name="WarningLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>WARNING: Will make log files in the same location where the logfilemaker is.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>HowManyFilesLabel</zorder>
  <zorder>TitleDescription</zorder>
  <zorder>numFilesSlider</zorder>
  <zorder>numFilesCurValLabel</zorder>
  <zorder>clearInputsBtn</zorder>
  <zorder>deleteLogFilesBtn</zorder>
  <zorder>submit_button</zorder>
  <zorder>statusText</zorder>
  <zorder>statusValueLabel</zorder>
  <zorder>WarningLabel</zorder>
  <zorder>LogFileMakerTitle</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
